rep-2000.rst

```diff
@@ -564,7 +564,7 @@ Dependency Requirements:
 +-------------+----------------+---------------+----------------+----------------+-------------------+
 | Connext DDS |                              5.3.1***           |               N/A                  |
 +-------------+----------------+---------------+----------------+----------------+-------------------+
-| Cyclone DDS |                                 0.5.1 (Eusebius)                                     |
+| Cyclone DDS |                                 0.7.x (Coquette)                                     |
```

Comment:
We haven't done the sync for Eloquent as of today, so this isn't quite available to the general public (except through the testing repositories).  I'm going to approve this, but not merge it until we do the final Eloquent sync.